<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblCredit">
      <property name="text">
       <string>Выберите срок вклада</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rb_1_month">
      <property name="text">
       <string>1 Месяц</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rb_6_month">
      <property name="text">
       <string>6 Месяцев</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rb_12_month">
      <property name="text">
       <string>12 Месяцев</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
